src/game/utils/flop_helper.rs

```diff
@@ -0,0 +1,261 @@
+        "AcJc9c", "Jc9c2d", "Jc9c3d", "Jc9c4d", "Jc9c5d", "Jc9c6d", "Jc9c7d", "Jc9c8d", "Jc9c9d",
+        "JcTd9c", "JcJd9c", "QdJc9c", "KdJc9c", "AdJc9c", "KcQc9c", "AcQc9c", "Qc9c2d", "Qc9c3d",
+        "Qc9c4d", "Qc9c5d", "Qc9c6d", "Qc9c7d", "Qc9c8d", "Qc9c9d", "QcTd9c", "QcJd9c", "QcQd9c",
+        "KdQc9c", "AdQc9c", "AcKc9c", "Kc9c2d", "Kc9c3d", "Kc9c4d", "Kc9c5d", "Kc9c6d", "Kc9c7d",
+        "Kc9c8d", "Kc9c9d", "KcTd9c", "KcJd9c", "KcQd9c", "KcKd9c", "AdKc9c", "Ac9c2d", "Ac9c3d",
+        "Ac9c4d", "Ac9c5d", "Ac9c6d", "Ac9c7d", "Ac9c8d", "Ac9c9d", "AcTd9c", "AcJd9c", "AcQd9c",
+        "AcKd9c", "AcAd9c", "9c9d9h", "Th9c9d", "Jh9c9d", "Qh9c9d", "Kh9c9d", "Ah9c9d", "TdTh9c",
+        "JhTd9c", "QhTd9c", "KhTd9c", "AhTd9c", "JdJh9c", "QhJd9c", "KhJd9c", "AhJd9c", "QdQh9c",
+        "KhQd9c", "AhQd9c", "KdKh9c", "AhKd9c", "AdAh9c", "QcJcTc", "KcJcTc", "AcJcTc", "JcTc2d",
+        "JcTc3d", "JcTc4d", "JcTc5d", "JcTc6d", "JcTc7d", "JcTc8d", "JcTc9d", "JcTcTd", "JcJdTc",
+        "QdJcTc", "KdJcTc", "AdJcTc", "KcQcTc", "AcQcTc", "QcTc2d", "QcTc3d", "QcTc4d", "QcTc5d",
+        "QcTc6d", "QcTc7d", "QcTc8d", "QcTc9d", "QcTcTd", "QcJdTc", "QcQdTc", "KdQcTc", "AdQcTc",
+        "AcKcTc", "KcTc2d", "KcTc3d", "KcTc4d", "KcTc5d", "KcTc6d", "KcTc7d", "KcTc8d", "KcTc9d",
+        "KcTcTd", "KcJdTc", "KcQdTc", "KcKdTc", "AdKcTc", "AcTc2d", "AcTc3d", "AcTc4d", "AcTc5d",
+        "AcTc6d", "AcTc7d", "AcTc8d", "AcTc9d", "AcTcTd", "AcJdTc", "AcQdTc", "AcKdTc", "AcAdTc",
+        "TcTdTh", "JhTcTd", "QhTcTd", "KhTcTd", "AhTcTd", "JdJhTc", "QhJdTc", "KhJdTc", "AhJdTc",
+        "QdQhTc", "KhQdTc", "AhQdTc", "KdKhTc", "AhKdTc", "AdAhTc", "KcQcJc", "AcQcJc", "QcJc2d",
+        "QcJc3d", "QcJc4d", "QcJc5d", "QcJc6d", "QcJc7d", "QcJc8d", "QcJc9d", "QcJcTd", "QcJcJd",
+        "QcQdJc", "KdQcJc", "AdQcJc", "AcKcJc", "KcJc2d", "KcJc3d", "KcJc4d", "KcJc5d", "KcJc6d",
+        "KcJc7d", "KcJc8d", "KcJc9d", "KcJcTd", "KcJcJd", "KcQdJc", "KcKdJc", "AdKcJc", "AcJc2d",
+        "AcJc3d", "AcJc4d", "AcJc5d", "AcJc6d", "AcJc7d", "AcJc8d", "AcJc9d", "AcJcTd", "AcJcJd",
+        "AcQdJc", "AcKdJc", "AcAdJc", "JcJdJh", "QhJcJd", "KhJcJd", "AhJcJd", "QdQhJc", "KhQdJc",
+        "AhQdJc", "KdKhJc", "AhKdJc", "AdAhJc", "AcKcQc", "KcQc2d", "KcQc3d", "KcQc4d", "KcQc5d",
+        "KcQc6d", "KcQc7d", "KcQc8d", "KcQc9d", "KcQcTd", "KcQcJd", "KcQcQd", "KcKdQc", "AdKcQc",
+        "AcQc2d", "AcQc3d", "AcQc4d", "AcQc5d", "AcQc6d", "AcQc7d", "AcQc8d", "AcQc9d", "AcQcTd",
+        "AcQcJd", "AcQcQd", "AcKdQc", "AcAdQc", "QcQdQh", "KhQcQd", "AhQcQd", "KdKhQc", "AhKdQc",
+        "AdAhQc", "AcKc2d", "AcKc3d", "AcKc4d", "AcKc5d", "AcKc6d", "AcKc7d", "AcKc8d", "AcKc9d",
+        "AcKcTd", "AcKcJd", "AcKcQd", "AcKcKd", "AcAdKc", "KcKdKh", "AhKcKd", "AdAhKc", "AcAdAh",
+    ]
+    .iter()
+    .map(|flop_str| flop_from_str(flop_str).unwrap())
+    .collect();
+}
+
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub enum Texture {
```

Comment:
Are we using this anywhere? I don't know that we need texture stuff just yet. Might not hurt to include but I'd prefer to not use it unless we are using it for a feature ATM

Reply:
Using it within the example for report generation. Happy to remove if you think it makes more sense to explicitly list the flops there.

Reply:
Okay, happy to keep this in an example if we think it's useful. Probably not part of the library for now, but we can think about supporting this in the future.

I generally want to keep things simple for now and be wary of adding new stuff unless we need it, which is why I'm being cautions about adding things.

Reply:
Removed for now